src/experiments/rl_envs/JuliaRL_SAC_Pendulum.jl

```diff
@@ -76,9 +77,16 @@ function RLCore.Experiment(
     hook = ComposedHook(
         total_reward_per_episode,
         time_per_step,
-        DoEveryNEpisode() do t, agent, env
+        DoEveryNStep() do t, agent, env
             with_logger(lg) do
-                @info "training" reward = total_reward_per_episode.rewards[end]
+                @info(
+                    "training",
+                    reward_term = agent.policy.reward_term,
+                    entropy_term = agent.policy.entropy_term,
+                )
+                if is_terminated(env)
+                    @info "training" reward = total_reward_per_episode.reward log_step_increment = 0
```

Comment:
If I use `total_reward_per_episode.rewards[end]` here it errors since the reward is added to the vector in the PostEpisodeStage while the DoEveryNStep is called in the PostActStage. It works for PPO_Pendulum because there it is using TotalBatchRewardPerEpisode which does both the collecting of rewards and the termination check in the PostActStage. Just seemed a bit odd and inconsistent to me so thought I would note it.